src/python/grpcio/grpc/experimental/aio/_call.py

```diff
@@ -0,0 +1,245 @@
+# Copyright 2019 gRPC authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Invocation-side implementation of gRPC Asyncio Python."""
+import asyncio
+import enum
+from typing import Callable, Dict, Optional, ClassVar
+
+import grpc
+from grpc import _common
+from grpc._cython import cygrpc
+
+DeserializingFunction = Callable[[bytes], str]
+
+
+class AioRpcError(grpc.RpcError):
+    """An RpcError to be used by the asynchronous API."""
+
+    _code: grpc.StatusCode
+    _details: Optional[str]
+    _initial_metadata: Optional[Dict]
+    _trailing_metadata: Optional[Dict]
+
+    def __init__(self,
+                 code: grpc.StatusCode,
+                 details: Optional[str] = None,
+                 initial_metadata: Optional[Dict] = None,
+                 trailing_metadata: Optional[Dict] = None):
+        super().__init__(self)
+        self._code = code
+        self._details = details
+        self._initial_metadata = initial_metadata
+        self._trailing_metadata = trailing_metadata
+
+    def code(self) -> grpc.StatusCode:
+        """
```

Comment:
This is more about Google's convention that the docstring should follow certain pattern. Can you take a look at http://google.github.io/styleguide/pyguide.html#384-classes? This apply to other docstrings in this file.

Reply:
thanks!